test/framework/kogitojobsservice.go

```diff
@@ -33,7 +33,7 @@ func InstallKogitoJobsService(namespace string, installerType InstallerType, rep
 	GetLogger(namespace).Infof("%s install Kogito Jobs Service with %d replicas and persistence %v", installerType, replicas, persistence)
 	switch installerType {
 	case CLIInstallerType:
-		return cliInstallKogitoJobsService(namespace, replicas, persistence)
+		return cliInstallKogitoJobsService(namespace, persistence)
```

Comment:
why did you remove the `replicas` here ?

Reply:
nevermind, created https://github.com/ricardozanini/kogito-cloud-operator/pull/9 for you :)

Reply:
I've removed because Go lint was screaming at me that the argument wasn't being used. :)

Now that you put some usage for it, we can work on it. :)